concepts/peg-stability-mechanisms.md

```diff
@@ -1,5 +1,35 @@
-# TBC
+# Peg Stability Mechanisms
 
-{% hint style="info" %}
-Sorry! We haven't been able to publish documentation on this topic yet. Please check back soon.
-{% endhint %}
+## Introduction
+
+The Endgame Plan allows the Maker Protocol to dynamically impact the supply and demand of DAI. These mechanisms are called Peg Stability Mechanisms. Note that should not be confused with the similarly-named [Peg Stability Modules (PSMs)](https://manual.makerdao.com/module-index/module-psm)
+
+{% hint style="warning" %} This documentation describes planned functionality and processes that MakerDAO has not yet implemented. Be aware that parts may be inaccurate or out of date. {% endhint %}
+
+## DAI Savings Rate (DSR)
+
+The [DAI Savings Rate](https://manual.makerdao.com/parameter-index/core/param-dai-savings-rate) already exists in Maker and remains unchanged in Endgame. It allows DAI holders to deposit DAI into the DSR contract and earn interest. The interest is paid from the protocol's profits. 
+
+By changing the interest rate, Maker Governance can increase or decrease the circulating supply of DAI, which in turn influences the price of DAI on the open market.
+
+## Stability Fee Base Rate (SBFR)
+
+The Stability Fee Base Rate increases the stability fee of some or all vaults, increasing income to help fund the DSR. A related and similar parameter that already exists in Maker is the [Global Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee#considerations) parameter.
+
+By changing the stability fee base rate, Maker Governance can make it more or less profitable to mint DAI using vaults. This increases or decreases the circulating supply of DAI, which in turn influences the price of DAI on the open market.
+
+## Target Rate (TR)
```

Comment:
Maybe needs more detail in this section, as it's one of the more unfamiliar concepts to the audience.

Reply:
Also should maybe highlight the long-term impact of using this lever.

Reply:
I've addressed this now but is worth a second look/chat before we sign off on it